backend/data/final_data/conditionsProcessed.json

```diff
@@ -4679,11 +4679,11 @@
     },
     "DPBS1100": {
         "original": "UNSW Global Diplomas only (7001, 7002, 7003, 7004)<br/><br/>",
-        "processed": "UNSW Global Diplomas only (7001, 7002, 7003, 7004)"
+        "processed": "(7001 || 7002 || 7003 || 7004)"
```

Comment:
Do we need to include some kind of note that you need to be enrolled in a UNSW Global Diploma or is that handled already by something else?

Reply:
oh nvm those are programs being listed, not courses